apps-rendering/src/components/headline.tsx

```diff
@@ -153,12 +177,57 @@ const getStyles = (format: ArticleFormat): SerializedStyles => {
 	}
 };
 
-const Headline = ({ item }: Props): ReactElement => (
-	<h1 css={getStyles(item)}>
-		<span>{item.headline}</span>
-		<StarRating item={item} />
-	</h1>
-);
+const headlineStyles = (format: ArticleFormat): SerializedStyles => {
+	return css`
+		color: ${text.headline(format)};
+		background-color: ${neutral[0]};
```

Comment:
Can this live in the editorial palette? There's a `background.headline` function there that we can add an interview `case` to.